### DISCUSSION TITLE
2fa

### DISCUSSION THREAD

**Discussion by Stephen-Ridgway** [Security-Onion-Solutions/securityonion#3840] [Unsupported Versions] [answered]
Hi!

Has anyone configured 2fa (Google Authenticator or other similar) for Security Onion 2? If so, could you point me in the direction of some instructions?

Kind regards,
Stephen

**Comment by weslambert** [comment, marked as answer]
You should be able to configure it for the OS using standard procedures (Ex. https://www.digitalocean.com/community/tutorials/how-to-set-up-multi-factor-authentication-for-ssh-on-centos-7).  With regard to the web interface, this is on the roadmap.
